src/main/scala/tech/cryptonomic/conseil/util/DatabaseUtil.scala

```diff
@@ -38,15 +38,18 @@ object DatabaseUtil {
       * @param  xs list of values to be inserted into SQLAction
       * @return SqlActionBuilder with values from parameter
       */
-    def insertValuesIntoSqlAction[T](xs: Seq[T]): SQLActionBuilder = {
-      var b = sql"("
-      var first = true
-      xs.foreach { x =>
-        if (first) first = false
-        else b = concatenateSqlActions(b, sql",")
-        b = concatenateSqlActions(b, sql"'#$x'")
+    def insertValuesIntoSqlAction[T](values: Seq[T]): SQLActionBuilder = {
```

Comment:
please change the param name in the javadoc

Reply:
sure, done, nice catch